Add University Workshops

monicaignatb opened the pull request:
Added:
Software Defined Instrumentation
Introduction to Electronics
Embedded Baremetal - Theoretical Part

gastmaier commented:
Hi Monica, push review commits to https://github.com/analogdevicesinc/documentation/tree/monica-main because I wasn't able to your fork.

Here are the changes;

* Removed trailing white-space
* Removed figures labels, since they are global and cannot be duplicated across pages*,
* Dokuwiki, ez, git links I used the proper [roles](https://analogdevicesinc.github.io/doctools/docs_guidelines/roles.html) (they allow us to automate link testing)
* Replace the table to put images side-by-side by a [grid](), same look, easier to maintain/create;


* They are totally ok to use when referenced on the text, see https://analogdevicesinc.github.io/documentation/university/active_learning/wien_bridge_oscillator/index.html, https://github.com/analogdevicesinc/documentation/blob/main/docs/university/active_learning/wien_bridge_oscillator/index.rst?plain=1 on how to reference them.

If you are ok with the changes, please squash my commits into yours and force push here, thanks!

monicaignatb commented:
Hi Jorge,

Thank you for the review, I wonder why you were not able to push the changes into my fork?
I don’t think I have rights to push to the analogdevices github.
And I also noticed the way the documentation now looks in the new branch, it seems like the grid is not recognized:
***@***.***
How can we fix this?

Thank you,
Monica
From: Jorge Marques ***@***.***>
Sent: Thursday, February 6, 2025 7:45 PM
To: analogdevicesinc/documentation ***@***.***>
Cc: Ignat, Monica ***@***.***>; Author ***@***.***>
Subject: Re: [analogdevicesinc/documentation] Add University Workshops (PR #16)

[External]


Hi Monica, I pushed review commits to https://github.com/analogdevicesinc/documentation/tree/monica-main<https://urldefense.com/v3/__https:/github.com/analogdevicesinc/documentation/tree/monica-main__;!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBMruF4vGk$> because I wasn't able to your fork.

Here are the changes;

  *   Removed trailing white-space
  *   Removed figures labels, since they are global and cannot be duplicated across pages*,
  *   Dokuwiki, ez, git links I used the proper roles<https://urldefense.com/v3/__https:/analogdevicesinc.github.io/doctools/docs_guidelines/roles.html__;!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBMZ3Cmmgc$> (they allow us to automate link testing)
  *   Replace the table to put images side-by-side by a grid, same look, easier to maintain/create;
  *   They are totally ok to use when referenced on the text, see https://analogdevicesinc.github.io/documentation/university/active_learning/wien_bridge_oscillator/index.html<https://urldefense.com/v3/__https:/analogdevicesinc.github.io/documentation/university/active_learning/wien_bridge_oscillator/index.html__;!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBM7zqzNgs$>, https://github.com/analogdevicesinc/documentation/blob/main/docs/university/active_learning/wien_bridge_oscillator/index.rst?plain=1<https://urldefense.com/v3/__https:/github.com/analogdevicesinc/documentation/blob/main/docs/university/active_learning/wien_bridge_oscillator/index.rst?plain=1__;!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBM8jy0OL0$> on how to reference them.

If you are ok with the changes, please squash my commits into yours and force push here, thanks!

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/analogdevicesinc/documentation/pull/16*issuecomment-2640572252__;Iw!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBMLstCyTQ$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AWMQ3J26GDTZPHGWVG2ZUIT2OONQVAVCNFSM6AAAAABWTWBPZOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNBQGU3TEMRVGI__;!!A3Ni8CS0y2Y!-Bk3k0BMZ5IpZtAPjaRWRxhVsofTD_tByKdCOBP67nV83FE3WaZO5-Wn1_hu_B5rNAMz18n9hiYOi5_8rpBMOHdMP0o$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
